loadtest/control/gencontroller/actions.go

```diff
@@ -132,21 +133,32 @@ func (c *GenController) createDirectChannel(u user.User) (res control.UserAction
 		return control.UserActionResponse{Err: control.NewUserError(err)}
```

Comment:
Do we need to perhaps adjust the fetch size in the previous line to ensure we have enough users in store or is it covered somewhere else?

Reply:
Great point! Maybe we can call it with the maximum number of users, since this number shouldn't be too large in practice. Thoughts?

Reply:
That sounds reasonable to me :+1:

Reply:
Just make sure we don't hit a page limit server side or we may have to page through.

Reply:
As far as I can see, there is no server-side limit for this endpoint. The perPage parameter is the one used.

Reply:
We're limited by the `memstore.Config.MaxStoredUsers`, though. That's right now set to 500, so it's exactly what we need, but maybe we should rethink all this in the future.